apps/directory/src/components/report-components/FactsTable.vue

```diff
@@ -363,7 +363,7 @@ export default {
           next = next[trailPart];
         }
       }
-      return value ?? "Unknown";
+     return value ?? "Unknown";
```

Comment:
Don't forget to run the code formatter :)

Reply:
You can enable it to run automatic (but might not work on Windows) as described here https://molgenis.github.io/molgenis-emx2/#/molgenis/dev_quickstart?id=pre-commit-hook

Reply:
just running 'gradle build' or 'gradle format' will also run the formatter :)